lib/utils.pm

```diff
@@ -1156,11 +1154,32 @@ sub reconnect_s390 {
     }
     else {
         my $r = wait_serial($login_ready, 300);
-        if ($r =~ qr/Welcome to SUSE Linux Enterprise 15/) {
-            record_soft_failure('bsc#1040606');
-        }
-        elsif (is_sle) {
-            $r =~ qr/Welcome to SUSE Linux Enterprise Server/ || die "Correct welcome string not found";
+        if (is_sle) {
```

Comment:
The below all looks great but better not hide it in "is_sle". I guess the only SLE-specific part is the welcome message itself.